lms/djangoapps/teams/static/teams/js/spec/views/my_teams_spec.js

```diff
@@ -41,36 +32,37 @@ define([
         });
 
         it('shows a message when the user is not a member of any teams', function () {
-            var teamMemberships = TeamSpecHelpers.createMockTeamMemberships([]),
-                myTeamsView = createMyTeamsView({
-                    teams: teamMemberships,
-                    teamMemberships: teamMemberships
-                });
+            var teams = TeamSpecHelpers.createMockTeams({results: []}),
+                myTeamsView = createMyTeamsView(teams);
             TeamSpecHelpers.verifyCards(myTeamsView, []);
             expect(myTeamsView.$el.text().trim()).toBe('You are not currently a member of any team.');
         });
 
         it('refreshes a stale membership collection when rendering', function() {
             var requests = AjaxHelpers.requests(this),
-                teamMemberships = TeamSpecHelpers.createMockTeamMemberships([]),
-                myTeamsView = createMyTeamsView({
-                    teams: teamMemberships,
-                    teamMemberships: teamMemberships
-                });
+                teams = TeamSpecHelpers.createMockTeams({
+                    results: []
+                }, {
+                    per_page: 2,
+                    url: TeamSpecHelpers.testContext.teamMembershipDetailUrl,
+                    username: TeamSpecHelpers.testContext.userInfo.username
+                }, MyTeamsCollection),
+                myTeamsView = createMyTeamsView(teams);
             TeamSpecHelpers.verifyCards(myTeamsView, []);
             expect(myTeamsView.$el.text().trim()).toBe('You are not currently a member of any team.');
-            teamMemberships.teamEvents.trigger('teams:update', { action: 'create' });
+            TeamSpecHelpers.teamEvents.trigger('teams:update', { action: 'create' });
             myTeamsView.render();
             AjaxHelpers.expectRequestURL(
                 requests,
-                TeamSpecHelpers.testContext.teamMembershipsUrl,
+                TeamSpecHelpers.testContext.teamMembershipDetailUrl,
                 {
-                    expand : 'team,user',
+                    expand : 'user',
                     username : TeamSpecHelpers.testContext.userInfo.username,
                     course_id : TeamSpecHelpers.testContext.courseID,
                     page : '1',
-                    page_size : '10',
-                    text_search: ''
+                    page_size : '2',
```

Comment:
Why did you decrease the page size? It shouldn't matter, right? Unless you are testing something about page sizes, it would seem clearer to leave it the same default as the feature uses.